cmd/destroy/cmddestroy.go

```diff
@@ -43,7 +43,8 @@ func NewCmdDestroy(f util.Factory, ioStreams genericclioptions.IOStreams) *cobra
 	cmdutil.AddValidateFlags(cmd)
 	_ = cmd.Flags().MarkHidden("validate")
 
-	cmd.Flags().BoolVar(&destroyer.DryRun, "dry-run", destroyer.DryRun, "If true, only print the objects that would be deleted, without performing it.")
+	cmd.Flags().BoolVar(&destroyer.DryRun, "dry-run", destroyer.DryRun, "NOT USED")
```

Comment:
Do we need to set the flag at all? If yes, just add a comment here. It looks strange that we create the flag and then immediately hide it.